nilearn/datasets.py

```diff
@@ -176,7 +176,7 @@ def _chunk_read_(response, local_file, chunk_size=8192, report_hook=None,
     return
 
 
-def _get_dataset_dir(dataset_name, data_dir=None, env_vars=[],
+def _get_dataset_dir(dataset_name, data_dir=None, pre_dirs=[],
```

Comment:
maybe dirs_to_search_first, or paths_to_explore_first, priority_searched_paths, any combination of these, or something even better ...

By the way default mutable argument is not recommended.